### ISSUE TITLE
Strontium Clathrates does not appear in Market list

### ISSUE THREAD

**Issue by ullerrm** [pyfa-org/Pyfa#600]
It's really handy to be able to drag Stront into a ship's cargo bay tab, for use with entosis modules.  However, it does not show up on a search.  Having it under Ammunition & Charges would be ideal.

**Comment by blitzmann** [comment]
A long while ago someone asked about adding isotopes. After looking into it and the market code, I decided against it at the moment. However, you stating to just add it to the existing Ammunition & Charges category... that may just work. I believe I could fairly easily force those specific item to their own category. I'll have to look into it a bit more =)

**Comment by MrNukealizer** [comment]
Yeah, now that you can export the fit via CREST and buy all, it would definitely be handy.

**Comment by blitzmann** [comment]
Implemented what I hope is a good enough compromise. Adding them to the market list itself is kinda ehh, but the items do show up now when you search for them. I also made it easier to include new groups of items in the search results should the need arise (ie: other modules that don't really belong in the market list, but folks may want to carry around). Open to suggestion on other things that might be good to add.
